src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```

Reply:
fixed in comment 9cc75945e6c3c7b96a493827f76c4bb2c96ebdca